### DISCUSSION TITLE
Parse with space between commas/quotes

### DISCUSSION THREAD

**Discussion by sagicoh1003** [JoshClose/CsvHelper#1920] [Q&A] [answered]
Hey

I'm parsing a csv file that looks like this: "Header1","header2"
"Field1", "field2"

Im getting an exception due to the space after the comma.

I wanted to ask if there is a way/configuration to not get an exception and to parse the field correctly? Field1 under header1 and so on..

Thanks

This is the stackoverflow link https://stackoverflow.com/questions/70259202/csvhelper-space-between-quotes

**Comment by JoshClose** [comment, marked as answer]
You can set `TrimOptions` in the configuration.

```cs
var config = new CsvConfiguration(CultureInfo.InvariantCulture)
{
    TrimOptions = TrimOptions.Trim
};
```
